code/modules/mob/dead/observer/observer.dm

```diff
@@ -1213,6 +1213,9 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 
 	. += ""
 
+	if(timeofdeath)
+		. += "Time Since Death: [duration2text_sec(world.time - timeofdeath)]"
+
```

Comment:
Move this either before or after the larva queue message below. I don't think hijack/evac/sd messages should follow this.

Reply:
Done!
**Before:**
![before](https://github.com/cmss13-devs/cmss13/assets/57483089/2e98eca6-45cd-4a9a-bf8b-bf761421a85b)

**After:**
![after](https://github.com/cmss13-devs/cmss13/assets/57483089/9df4bba6-eac6-45f9-8e17-7be515c42191)